test/libsolidity/syntaxTests/array/uninitialized_storage_var.sol

```diff
@@ -1,9 +0,0 @@
-contract C {
```

Comment:
I removed this file because, after removing the check, this file will give the following three warnings:

// Warning: (38-54): Unused local variable.
// Warning: (58-76): Unused local variable.
// Warning: (14-80): Function state mutability can be restricted to pure

There are other testcases that deals with these warnings, so this case is now redundant.

Reply:
You could add just add ``x; y;`` to this, then it might make sense to keep it.